index.js

```diff
@@ -2,82 +2,91 @@
 var Cesium = require('cesium');
 var bodyParser = require('body-parser');
 var express = require('express');
-var gitHubWebHook = require('express-github-webhook');
 var Promise = require('bluebird');
 
 var defined = Cesium.defined;
 
+var checkWebHook = require('./lib/checkWebHook');
 var commentOnClosedIssue = require('./lib/commentOnClosedIssue');
 var commentOnOpenedPullRequest = require('./lib/commentOnOpenedPullRequest');
 
 var dateLog = require('./lib/dateLog');
 var Settings = require('./lib/Settings');
 
 Settings.loadRepositoriesSettings('./config.json')
-.then(function (repositoryNames) {
-    dateLog('Loaded settings successfully');
-    var webHookHandler = gitHubWebHook({
-        path: Settings.listenPath,
-        secret: Settings.secret
-    });
+    .then(function (repositoryNames) {
+        dateLog('Loaded settings successfully');
+
+        var app = express();
 
-    var app = express();
-    app.use(bodyParser.json());
-    app.use(webHookHandler);
+        app.use(bodyParser.json());
```

Comment:
`app.use` means a function runs on every route, not just routes we care about.  Rather passing `bodyParser.json` and `checkWebHook` to `app.use`, pass them as parameters to the `post` call below, this way they only run on the routes we actually care about.